code/modules/projectiles/guns/energy.dm

```diff
@@ -20,6 +20,11 @@
 	/// Do you want the gun to fit into a turret, defaults to true, used for if a energy gun is too strong to be in a turret, or does not make sense to be in one.
 	var/can_fit_in_turrets = TRUE
 
+/obj/item/gun/energy/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>It is [(cell.charge / cell.maxcharge) * 100]% charged.</span>"
```

Comment:
`/obj/item/gun/energy/plasmacutter/examine` has a similar examine proc, and it would display the cell charge line twice when examined. I would remove the plasma cutter proc and put what the plasmacutter has, into this proc:
```dm
/obj/item/gun/energy/examine(mob/user)
	. = ..()
	if(cell)
		. += "<span class='notice'>It is is [round(cell.percent())]% charged.</span>"
```